libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailManagementDialog/useScheduledEmailManagement.ts

```diff
@@ -1,48 +1,58 @@
 // (C) 2022 GoodData Corporation
 
+import { IScheduledMail } from "@gooddata/sdk-backend-spi";
 import {
     GoodDataSdkError,
     useBackendStrict,
     useCancelablePromise,
     useWorkspaceStrict,
 } from "@gooddata/sdk-ui";
-import {
-    selectCanManageScheduledMail,
-    selectCurrentUser,
-    selectDashboardRef,
-    useDashboardSelector,
-} from "../../../model";
+import { selectCanManageScheduledMail, selectDashboardRef, useDashboardSelector } from "../../../model";
 
 interface IUseScheduledEmailManagementProps {
     /**
      * Callback to be called, when emails fail to load.
      */
     onError?: (error: GoodDataSdkError) => void;
+
+    /**
+     * Callback to be called, when emails load.
+     */
+    onSuccess: (scheduledEmails: IScheduledMail[]) => void;
+
+    /**
+     * Flag to handle data refetching logic
+     */
+    needsReload: boolean;
```

Comment:
I would rename it to something like `loadScheduledMails` and place it on the top. It's used for "reload" from outside but within it's just "load"